### PR TITLE
Update pl.ts

### PR DESCRIPTION
**Pull request by atlochowski**

### CODE REVIEW

File: translations/pl.ts [hunk 1]
```diff
@@ -644,7 +644,7 @@
     <message>
         <source>Map download error</source>
         <extracomment>notification summary</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>podsumowanie powiadomienia</translation>
```
Comment (Karry) [inline translations/pl.ts:647]: "Map download error" is the source :-) "notification summary" is comment for translators... https://github.com/Karry/osmscout-sailfish/blob/master/qml/pages/MapPage.qml#L89

Comment (atlochowski) [reply translations/pl.ts:647]: sorry

Comment (atlochowski) [reply translations/pl.ts]: fixed

Comment (Karry) [reply translations/pl.ts]: no problem :-)

**Comment by Karry** [comment]
Thank you

**Pull request merged**